test/run

```diff
@@ -3,6 +3,15 @@
 # This is the expected entry point for Cockpit CI; will be called without
 # arguments but with an appropriate $TEST_OS, and optionally $TEST_SCENARIO
 
+# Currently supported scenarios:
+# devel       - runs tests with coverage enabled and generates a html file with coverage information
```

Comment:
... and `NODE_ENV=development` to get additional React checks and useful stack traces